README.md

```diff
@@ -172,20 +172,20 @@ const handleChange = (ev: React.MouseEvent<HTMLDivElement>) => { ... }
 
 # Component Typing Patterns
 
-## Stateless Components - SFC
+## Function Components - FC
 
-#### - stateless counter
+#### - functional counter
```

Comment:
functional counter is a bit confusing name, let's name it "counter component"

Reply:
sure, sounds good. I just changed it as stateless is no longer the term being used. are you happy for me to do it in this PR, as I said I was having issues with the instructions in the contributing guidelines - it wasn't working  😯

Reply:
Yea sure.
Could you also post what was the issue with contributing guide as I would like to fix that.
Thanks